starter/src/app/_components/home/filters.tsx

```diff
@@ -0,0 +1,70 @@
+import Results from "@/app/_components/home/results";
+import SidebarItem from "@/app/_components/home/sidebar-item";
+import SignupCard from "@/app/_components/home/signup-card";
+import { Layout } from "@/app/_components/universal/layout";
+import { sidebarCategories, resultItems } from "@/app/constants";
+import type { SidebarCategory } from "@/app/constants";
+import { Button } from "@/components/ui/button";
+import {
+  Dialog,
+  DialogPortal,
+  DialogOverlay,
+  DialogClose,
+  DialogTrigger,
+  DialogContent,
+} from "@/components/ui/dialog";
+
+export const HomeFilters = () => {
+  const isMd = false; // TODO: add hook to detect screen size
```

Comment:
this looks cool for tailwind-compatible breakpoints https://github.com/kodingdotninja/use-tailwind-breakpoint

Reply:
nice!